estate/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "Real Estate",
+    "summary": "Test module",
+    "version": "18.0",
+    "license": "OEEL-1",
+    "depends": ["crm"],
```

Comment:
I don't see any reason to make `crm`a dependency of your module.

```suggestion
    "depends": ["base"],
```